backend/src/controllers/admin/deleteCourse.ts

```diff
@@ -0,0 +1,19 @@
+import { NextFunction, Request, Response } from 'express';
+
+import { checkNotDeleteCoursesProvider } from 'db/providers/clientCourseProvider';
+import { deleteCourseProvider } from 'db/providers/courseProvider';
+
+const deleteCourse = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const courseId = req.params.id;
```

Comment:
```suggestion
    const { id: courseId } = req.params;
```
Let's use common project style of code writing